src/test/java/uk/gov/justice/digital/hmpps/keyworker/integration/ReferenceDataIntegrationTest.kt

```diff
@@ -25,7 +28,7 @@ class ReferenceDataIntegrationTest : IntegrationTest() {
 
   @ParameterizedTest
   @ValueSource(strings = [Roles.KEYWORKER_RO, Roles.KEYWORKER_RW])
-  fun `200 ok - can retrieve reference data with correct role`(role: String) {
+  fun `200 ok - can retrieve keyworker status`(role: String) {
```

Comment:
Probably worth tests for the two new RD domains. I don't think the UI will ever retrieve them but it would prove the API contract works as expected

Reply:
Unless I've missed that there already are tests of course :)